community: Add bio for Gunnar Stefansson (#5)
open-education-hub/open-education-hub-site: website/docs/community/people.md

# Project Team

> This is content

Reviewer: ```suggestion
```

![Gunnar Stefansson](../../static/img/people/gunnar_stefansson.png)

> Gunnar supervises the Icelandic component of the OpenEdu project, in particular the linking and format consistency with the tutor-web.

Reviewer: ```suggestion
Hi, I'm Gunnar.
I supervise the Icelandic component of the OpenEdu project, in particular the linking and format consistency with the tutor-web.
```

> Gunnar is a Professor of Statistics at the University of Iceland, Department of Mathematics.

Reviewer: ```suggestion
I am a Professor of Statistics at the University of Iceland, Department of Mathematics.
```
